LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-fr-expected.html

```diff
@@ -70,72 +96,52 @@
 
 <grid>
   <item>1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
-  <item style="height:4ch; grid-area: 2/1/4/auto">5 5</item>
-
-  <item class="hidden" style="grid-area: 2/1">0 0</item>
-  <item class="hidden" style="grid-area: 3/2">0 0</item>
+  <item style="height:4ch; grid-row: 2 / span 2; grid-column: 1;">5 5</item>
 </grid>
 
 <grid>
   <item style="grid-row: 4">1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
   <item style="grid-area: 2/1/4/2">5 5</item>
   <item style="height:5ch; grid-area: 1/2/4/3">6</item>
-
-  <item class="hidden" style="grid-area: 3/3">0</item>
-  <item class="hidden" style="grid-area: 2/1">0 0</item>
-  <item class="hidden" style="grid-area: 3/1">0 0</item>
 </grid>
 
 <grid>
   <item style="grid-row: 4">1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
-  <item style="height:3ch; grid-area: 2/1/4/2">5 5</item>
+  <item style="height:3ch; grid-area: 2/1/4/2">5</item>
   <item style="height:5ch; grid-area: 1/2/4/3">6</item>
-
-  <item class="hidden" style="grid-area: 2/1">0 0</item>
-  <item class="hidden" style="grid-area: 3/1">0 0</item>
 </grid>
 
 <grid>
   <item>1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
-  <item style="grid-area: 1/3/5/4">5 5</item>
-
-  <item class="hidden" style="grid-area: 1/3">0 0</item>
-  <item class="hidden" style="grid-area: 3/3">0 0</item>
+  <item style="grid-row: span 4;">5 5</item>
 </grid>
 
 <grid>
   <item>1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
-  <item style="height:6ch; grid-area: 1/3/5/4">5 5</item>
-
-  <item class="hidden" style="grid-area: 1/3">0 0</item>
-  <item class="hidden" style="grid-area: 4/3">0 0</item>
```

Comment:
I think it's fine since these items are hidden and are just there to influence track sizing? You may need to double check that the actual item(s) that are placed in the same track are placed explicitly in case they end up moving due to the hidden items being forced there